VQ10/README.TXT

```diff
@@ -0,0 +1,78 @@
+# codec2_VQ10
+Experiment tool for new Vector Quantizer for use in Codec2, 
+
+this version 2.0 implements several changes to the start version:
+- new options and functionality of c2sim
+- new learning speech data  prepared from TSP Lab samples
+- new EMBSD quality measurement for additional check
+- Linde-Buzo-Gray algorithm for optimising the codebook
+
+NOTES:
+- from now lspvt.csv, lspvtopt.csv and codebook.csv files share the same format
+- you do not put any data to the files manually, it is provided by options
+- the changes has been made to c2sim.c, defines.h, phase.c, sine.c, sine.h
+
+to get new environment you need to rebuild the c2sim:
+
+cd codec2/build_linux
+make c2sim
+
+than you will need to compile embsd
+
+gcc -o embsd ../embsd/embsd.c -lm
+
+you should have codec2/VQ10 directory and there the raw speech samples for this branch are stored. we will use them
+to start new version of VQ10 you will perform a build of it using c2sim.
+to be sure youre going from scratch please
+
+sudo rm *.csv
+
+please prepare csv directory 
+
+mkdir csv
+
+than you can run script which will perform all steps for building the vq10 codebook
+I suggest to run it in background as it will take some time (for me it is between 1-2 hrs.)
+
+sudo ../VQ10/lbg.sh &> out.txt &
+
```

Comment:
Why do we need sudo privileges?

Reply:
> Why do we need sudo privileges?

I think it can run without sudo too, I am not sure how privileges in Linux work, I don't know whether those in script are sufficient, I need cp, mv, rm commands to work inside the script. I learn what I have to first and then improve...